src/config.js

```diff
@@ -32,13 +32,15 @@ export const largeGraphThreshold = 1000;
 export const flags = {
   sizewarning: {
     name: 'Size Warning',
-    description: 'Show a warning before rendering very large graphs',
+    description:
+      'Show a warning before rendering very large graphs (requires page refresh on browser)',
```

Comment:
`on browser` sounds a little odd to me. Suggestion:

```suggestion
      'Show a warning before rendering very large graphs (page reload required)',
```